services/core-web/src/components/modalContent/UpdateStatusGenerateLetterModal.js

```diff
@@ -44,10 +45,9 @@ export class UpdateStatusGenerateLetterModal extends Component {
     this.props.type === "AIA" ? (
       <IssuePermitForm
         initialValues={{
-          issue_date: this.props.noticeOfWork.proposed_start_date,
-          auth_end_date: this.props.noticeOfWork.proposed_end_date,
+          issue_date: this.props.draftAmendment.issue_date,
```

Comment:
Is this document only available after a draft permit is created?

Reply:
yes, It is the final step in issuing a permit where the inspector sets the issue_date, auth_end_date - previously it was being populated with the propsed_dates, but we now set the dates prior on the draft tab so they can be used on this form now.